### PR TITLE
Supported VMware vCenter versions in Provisioning

### PR DESCRIPTION
**Pull request by Lennonka**
VMware vCenter Server 8 hasn't been properly tested yet, so we should explicitly state which vCenter versions are supported in Foreman. See also: https://bugzilla.redhat.com/show_bug.cgi?id=2247345#c6

Originally, I wanted to create a RH KB article, but it's probably better to include it in the docs directly for upstream.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/con_prerequisites-for-vmware-provisioning.adoc [hunk 1]
```diff
@@ -3,8 +3,19 @@
...
 
+* A supported version of VMware vCenter Server.
+The following versions have been fully tested with {Project}:
+** vCenter Server 7.0
+** vCenter Server 6.7 (EOL)
+** vCenter Server 6.5 (EOL)
+ifndef::satellite[]
+
++
+vCenter Server 8.0.0 and 8.0.1 are known to work with {Project}, but have not been fully tested yet and, therefore, are not yet supported.
```
Comment (adamlazik1) [inline guides/common/modules/con_prerequisites-for-vmware-provisioning.adoc:14]:
```suggestion
vCenter Server 8.0.0 and 8.0.1 are known to work with {Project}, but have not been fully tested yet and are not officially supported.
```

**Review by adamlazik1 — COMMENTED** [review]
Content looks good.

File: guides/common/modules/con_prerequisites-for-vmware-provisioning.adoc [hunk 2]
```diff
@@ -3,8 +3,18 @@
...
+* A supported version of VMware vCenter Server.
+The following versions have been fully tested with {Project}:
+** vCenter Server 7.0
+** vCenter Server 6.7 (EOL)
+** vCenter Server 6.5 (EOL)
+ifndef::satellite[]
+
++
+vCenter Server 8.0.0 and 8.0.1 are known to work with {Project}, but have not been fully tested yet and are not officially supported.
+endif::[]
```
Comment (Lennonka) [inline guides/common/modules/con_prerequisites-for-vmware-provisioning.adoc:15]: @maximiliankolb Do you wish to keep this note for orcharhino?

**Review by chris1984 — APPROVED** [review]
Looks good

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by Lennonka** [comment]
Cherry-picked:
- 446dae773..6ab251459  3.9 -> 3.9
- c9f61a832..bed4627ec  3.8 -> 3.8
- 48c271d1c..fd7dfeb49  3.7 -> 3.7
- 91b41b345..69a89750a  3.6 -> 3.6
- 369e7874e..a2cd9b17e  3.5 -> 3.5
- f62c98656..9529c77a3  3.4 -> 3.4
- 989234cad..4c949a394  3.3 -> 3.3
- 8aed59ac2..98e9493e0  3.2 -> 3.2
- bafbccd40..4bff0a6d3  3.1 -> 3.1

**Pull request merged**